city_scrapers/spiders/chi_ssa_23.py

```diff
@@ -0,0 +1,143 @@
+import logging
+import re
+import unicodedata
+from datetime import datetime, timedelta
+
+from city_scrapers_core.constants import COMMISSION
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+
+
+class ChiSsa23Spider(CityScrapersSpider):
+    name = "chi_ssa_23"
+    agency = "Chicago Special Service Area #23 Clark Street"
+    timezone = "America/Chicago"
+    start_urls = ["https://www.lincolnparkchamber.com/clark-street-ssa-administration/"]
+    location = {
+        "name": "Lincoln Park Chamber of Commerce",
+        "address": "2468 N. Lincoln Chicago, IL 60614"
+    }
+    # Each meeting takes place on Wednesday 4 PM
+    meeting_day = 'Wednesday'
+    time = '4 pm'
+
+    def parse(self, response):
+        # Due to the current lack of documents for the meetings of 2020
+        # no assumption is made regarding the expected HTML page format
+        # once these are uploaded.
+        h4s = response.xpath('//h4')
+        meeting_years = list()
+
+        for entry in h4s.xpath('./text()').getall():
+            # Do not allow duplicates
+            if entry[0:4] not in meeting_years:
+                meeting_years.append(entry[0:4])
+
+        # General meeting description is mentioned just after the H4 for the current year
+        general_desc = h4s.xpath('following-sibling::p[1]//em//text()').extract_first()
+
+        # List containing all meeting dictionaries
+        meetings = list()
+
+        link_collection = []
+        for h4 in h4s[1:]:
+            # Get the links for each meeting
+            link_collection.append(h4.xpath('following-sibling::p[1]//a'))
```

Comment:
The `link_collection` setup here is a bit hard to understand. It might be easier to understand as dictionaries where the key is the meeting date and the value is a list of links?

Reply:
It took some time, but I got back to this today and changed quite much. The current version is based on dictionaries and seems to be more general.